.circleci/config.yml

```diff
@@ -568,7 +568,7 @@ jobs:
     executor: mac
     steps:
       - run-tests-mac:
-          test_targets: "other wasm0.test_lua skip:other.test_native_link_error_message skip:other.test_emcc_v"
+          test_targets: "other wasm0.test_lua wasm0.test_longjmp skip:other.test_native_link_error_message skip:other.test_emcc_v"
```

Comment:
Why disable this test ?

Reply:
It's not disabling a test - it's adding a new test that wasn't run before (there is no `skip:` on it)